src/api/api_auth.nit

```diff
@@ -45,9 +46,10 @@ class MissionsGithubOAuthCallBack
 		if session == null then return
 		var user = session.user
 		if user == null then return
-		var player = config.players.find_by_user(user)
+		var id = "github:" + user.login
```

Comment:
I see 3 solutions:

* ignore collision :/
* check them and ask for the player to choose another login for `missions` (and keep an `origin` field)
* totally change the id system :(